site/en/guide/migrate.ipynb

```diff
@@ -487,20 +484,19 @@
         "id": "BsAseSMfB9XN"
       },
       "source": [
-        "* The simple stack of layers fits neatly into `tf.keras.Sequential`. (For more complex models see [custom layers and models](keras/custom_layers_and_models.ipynb), and [the functional API](keras/functional.ipynb).)\n",
-        "* The model tracks the variables, and regularization losses.\n",
-        "* The conversion was one-to-one because there is a direct mapping from `v1.layers` to `tf.keras.layers`.\n",
+        "- The simple stack of layers fits neatly into `tf.keras.Sequential`. (For more complex models, check out the [custom layers and models](keras/custom_layers_and_models.ipynb) and [the functional API](keras/functional.ipynb) guides.)\n",
+        "- The model tracks the variables, and regularization losses.\n",
+        "- The conversion was one-to-one because there is a direct mapping from `v1.layers` to `tf.keras.layers`.\n",
         "\n",
         "Most arguments stayed the same. But notice the differences:\n",
         "\n",
-        "* The `training` argument is passed to each layer by the model when it runs.\n",
-        "* The first argument to the original `model` function (the input `x`) is gone. This is because object layers separate building the model from calling the model.\n",
-        "\n",
+        "- The `training` argument is passed to each layer by the model when it runs.\n",
+        "- The first argument to the original `model` function (the input `x`) is gone. This is because object layers separate building the model from calling the model.\n",
         "\n",
         "Also note that:\n",
         "\n",
-        "* If you were using regularizers of initializers from  `tf.contrib`, these have more argument changes than others.\n",
-        "* The code no longer writes to collections, so functions like `v1.losses.get_regularization_loss` will no longer return these values, potentially breaking your training loops."
+        "- If you were using regularizers of initializers from `tf.contrib`, these have more argument changes than others.\n",
```

Comment:
using regularizers or initializers

Reply:
Good catch. Let's also change it to present continuous tense:

```diff
Also note that:
- - If you were using regularizers of initializers from `tf.contrib`, these have more argument changes than others.
+ - If you are using regularizers or initializers from `tf.contrib`, these have more argument changes than others.
```